developer_docs/overview.md

```diff
@@ -86,4 +86,4 @@ If the error prevents further translation (e.g. the type of an object is now unk
 
 ## Getting Help
 
-While discussions within the associated GitHub issue are often sufficient, should you require more help do not hesitate to ask one of the other developers to add you to our slack: <pyccel.slack.com>
+While discussions within the associated GitHub issue are often sufficient, should you require more help do not hesitate to ask one of the other developers to add you to our Slack channel <pyccel.slack.com>.
```

Comment:
I think this line needs modifying to mention discord instead anyway

Reply:
True, but do we have a public channel on Discord yet? cc: @bauom

Reply:
the `GENERAL` category have 2 channels `general` and `random` these can be viewed by anyone.
should we rename for something else ?

Reply:
we maybe should write a bigger message in the `welcome` channel and that were we should link.
here is a permanent link to the welcome channel 
https://discord.gg/2Q6hwjfFVb

Reply:
It is automatic now them will have access to the public channels which now are 
GENERAL : `general`, `random`
MEETING ROOMS: `room1`, `room2`


There is a way to make access temporary if we want.

Reply:
This sounds good to me. I let you handle the server access as you see fit :wink: